infinispan-remote/src/main/java/org/hibernate/ogm/datastore/infinispanremote/impl/protobuf/SchemaDefinitions.java

```diff
@@ -64,7 +75,44 @@ public void deploySchema(String generatedProtobufName, RemoteCache<String, Strin
 			throw LOG.errorAtSchemaDeploy( generatedProtobufName, hrce );
 		}
 		if ( schemaCapture != null ) {
-			schemaCapture.put( generatedProtobufName, generatedProtoschema );
+			schemaCapture.put( generatedProtobufName, protoSchema );
+		}
+	}
+
+	private void checkCompatibility(String generated, String providedByService) {
+		String providedByServiceWithoutComments = providedByService.trim()
+				.replaceAll( "\\/\\*([\\S\\s]+?)\\*\\/", "" )
+				.replaceAll( "(?s)/\\*.*?\\*/", "" );
+
+		String generatedWithoutSpaces = generated.replaceAll( "\\s+", " " ).trim();
+		String providedByServiceWithoutCommentsAndSpaces = providedByServiceWithoutComments.replaceAll( "\\s+", " " ).trim();
+
+		if ( !generatedWithoutSpaces.equalsIgnoreCase( providedByServiceWithoutCommentsAndSpaces ) ) {
```

Comment:
@Sanne I don't like very much this solution... I'm going to figured out something better exploring Infinispan code.

Reply:
Cool. I'll stop the review then. Let me know when it's done.